Proxy/proxy.py

```diff
@@ -161,7 +162,8 @@ def testdb_read_worker():
     while(row is not None):
 
         port = row[1]
-        item = row[2]
+        item = {}
+        item["data"] = row[2]
         try:
             getDicts[unit][port].append(item)
```

Comment:
Yup and right here getDicts[unit][port] is correctly adding the data to a dictionary of dequeues for each station just like the normal Proxy. Looks good!